### PR TITLE
Request filters in query keys

### PR DESCRIPTION
**Pull request by Rendalf**

### CODE REVIEW

**Review by Lecsar — APPROVED** [review]

File: src/frontend.md [hunk 1]
```diff
@@ -421,6 +421,53 @@ export function fakeTicket() {}
    // no additional network request needed, use the first one
    ```
 
+4. Add query filters to query keys.
```
Comment (strashkov) [inline src/frontend.md:424]: not only filters, path parameters as well

Comment (Rendalf) [reply src/frontend.md:424]: > btw, what about https://tanstack.com/query/v4/docs/react/eslint/eslint-plugin-query ?

good idea, added a task into our backlog: https://github.com/OsomePteLtd/principles/pull/254

**Review by runnez — APPROVED** [review]

File: src/frontend.md [hunk 2]
```diff
@@ -421,6 +421,58 @@ export function fakeTicket() {}
    // no additional network request needed, use the first one
    ```
 
+4. Add query filters and path params to query keys.
```
Comment (strashkov) [inline src/frontend.md:424]: wouldn't you like to use the same words as here?
https://github.com/OsomePteLtd/principles/pull/254#discussion_r1267924640

**Review by strashkov — APPROVED** [review]

File: src/frontend.md (hunk 1 above)
Comment (Rendalf) [reply src/frontend.md]: yeah, will consider to remove it after eslint rule covers all cases

File: src/frontend.md (hunk 2 above)
Comment (Rendalf) [reply src/frontend.md]: To put it more clear, appended with this words.

**Review by Gashaal — APPROVED** [review]

**Pull request merged**